src/dotenv-vault.ts

```diff
@@ -0,0 +1,116 @@
+// Common Options
+const yesOption: Fig.Option = {
+  name: ["--yes", "-y"],
+  description:
+    "Automatic yes to prompts. Assume yes to all prompts and run non-interactively",
+};
+
+const helpOption: Fig.Option = {
+  name: "--help",
+  description: "Get documentation for the command",
+};
+
+const dotenvMeOption: Fig.Option = {
+  name: ["-m", "--dotenvMe"],
+  description:
+    "Pass .env.me (DOTENV_ME) credential directly (rather than reading from .env.me file)",
+  args: {
+    name: "DOTENV_ME",
+    description: "Pass .env.me (DOTENV_ME) credential directly",
+  },
+};
+
+// Common Arguments
+const dotenvVaultArg = {
```

Comment:
Absolutely not required but if you add the types you'll get nicer suggestions for the arg properties 🙂
```suggestion
const dotenvVaultArg: Fig.Arg = {
```

Reply:
I'll skip this for now.